src/lex/cpp.rs

```diff
@@ -1061,14 +1062,10 @@ impl<'a> PreProcessor<'a> {
     // now we want to do the dirty work of reading it into memory
     fn include_path(
         &mut self,
-        filename: Vec<u8>,
+        filename: String,
```

Comment:
`filename` should probably be an `std::path::PathBuf` so you get easier path semantics.